src/VisualStudio/Core/Def/Implementation/Library/FindResults/TreeItems/MetadataDefinitionTreeItem.cs

```diff
@@ -40,7 +40,7 @@ public override int GoToSource()
             if (symbol != null && referencingProject != null)
             {
                 var navigationService = _workspace.Services.GetService<ISymbolNavigationService>();
-                return navigationService.TryNavigateToSymbol(symbol, referencingProject, CancellationToken.None)
+                return navigationService.TryNavigateToSymbol(symbol, referencingProject, cancellationToken: CancellationToken.None)
```

Comment:
I think just because we have no cancellation token to pass, right? We could pop a threaded wait dialog, but since this is a programmatic API we can't really cancel if the caller wants it.